src/main/java/com/revlearn/team1/controller/TransactionController.java

```diff
@@ -0,0 +1,10 @@
+package com.revlearn.team1.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/transaction")
```

Comment:
I know this isnt your controller and its just so you don't get compile errors, but can we keep the RequestMapping convention consistent? either change /courses to /api/courses or api/transactions to /transactions